src/plugins/advanced_settings/public/index.ts

```diff
@@ -20,6 +20,10 @@ export { ComponentRegistry } from './component_registry';
 const LazyField = React.lazy(() => import('./management_app/components/field'));
 export { LazyField };
 
+export { toEditableConfig } from './management_app/lib/to_editable_config';
```

Comment:
Hum, I don't think this was ever meant for this internal utility to be publicly exposed. 

Does that mean that APM has its own page to display uiSetting-based settings, and/or that they're not displayed in the advanced settings admin section?

Reply:
Yes, APM will have a page to show its settings. But everything will still be available under Kibana advanced settings, this would be just a shortcut for the user.

Reply:
Ok. I guess exporting this will be fine until we move those to a dedicated utility package then.

In think you broke the `observability` bundle size somehow with this PR change though, given the last build's report

Reply:
See https://github.com/elastic/kibana/pull/139553#discussion_r957348740

Reply:
See https://github.com/elastic/kibana/pull/139553#discussion_r967191505